build-tools/packages/build-cli/src/library/commands/generateEntrypoints.ts

```diff
@@ -363,6 +411,17 @@ async function generateEntrypoints(
 			namedExports[namedExports.length - 1].trailingTrivia = "\n";
 		}
 
+		// legacy APIs do not accumulate to others
+		if (apiTagLevel !== "legacy") {
+			// Additionally, if beta should not accumulate to alpha (alpha may be
+			// treated specially such as mapped to /legacy) then skip beta too.
+			// eslint-disable-next-line unicorn/no-lonely-if
+			if (!separateBetaFromAlpha || apiTagLevel !== "beta") {
```

Comment:
Any way to refactor this to remove the double negative (not A or B != x)? `!A || !B` == `!(A && B)` so maybe

```suggestion
			if (!(separateBetaFromAlpha && apiTagLevel === "beta")) {
```

Though that's even more confusing to me... so maybe leave it. :) I'm not sure that's equivalent logically in this case anyway.

Reply:
I think we are okay with comment there.
Your replacement is equivalent. I tend to favor separable conditions over those that must be evaluated in conjuction (the revised version).